### PR TITLE
Feat/expense

### PR DESCRIPTION
**Pull request by Abhishek-440**
working on expense CRUD done

### CODE REVIEW

File: src/Routes/AppRoute.js [hunk 1]
```diff
@@ -10,6 +11,8 @@ function AppRoute() {
         <Route path="/Home" element={<Home />} />
         <Route path="/login" element={<Login />} />
         <Route path="/register" element={<Register />} />
+        <Route path="/expenseinput" element={<ExpenseInformation />} />
+        <Route path="/expensemonth" element={<ExpenseMonth />} />
```
Comment (rishav95) [inline src/Routes/AppRoute.js]: change path to /expense-input or /expense/input
same for month

File: src/components/organisms/Card/Card.js [hunk 2]
```diff
@@ -4,19 +4,24 @@ import { AiFillDelete, AiFillEdit } from "react-icons/ai";
...
   const { id, title, amount, date, created_at, updated_at } = data;
+  const navigate = useNavigate();
   // const expense = useSelector((state) => selectExpenseById(state, id));
   // const { title, amount, date, created_at } = expense;
   const dispatch = useDispatch();
 
   const onUpdate = () => {
     // dispatch(updateOldExpense(id));
-    <ExpenseInformation />;
+    navigate(`/expenseinput`, { state: data });
```
Comment (rishav95) [inline src/components/organisms/Card/Card.js]: change path to /expense-input or /expense/input

File: src/pages/Homepage.js [hunk 3]
```diff
@@ -29,48 +40,53 @@ function Home() {
...
-
-  // const incomeData = {
-  //   title: income.title,
-  //   amount: income.amount,
-  //   date: income.date,
-  //   created_at: income.created_at,
-  // };
+  //For adding more expenses
+  function expenseInformation() {
+    navigate("/expenseinput");
```
Comment (rishav95) [inline src/pages/Homepage.js]: change path to /expense-input or /expense/input

File: src/pages/Homepage.js [hunk 4]
```diff
@@ -89,16 +105,16 @@ function Home() {
...
+      <h1>EXPENSE</h1>
       <Row className="text-center mt-2">
-        <h1>EXPENSE</h1>
-        <ExpenseInformation
+        {/* <ExpenseInformation
           register={register}
           handleSubmit={handleSubmit}
           dispatch={dispatch}
           control={control}
           setValue={setValue}
```
Comment (rishav95) [inline src/pages/Homepage.js]: Remove commented-out code if its not going to be used

Comment (barunojha) [reply src/pages/Homepage.js]: Done!

File: src/pages/Homepage.js (hunk 3 above)
Comment (barunojha) [reply src/pages/Homepage.js]: Done!

File: src/components/organisms/Card/Card.js (hunk 2 above)
Comment (barunojha) [reply src/components/organisms/Card/Card.js]: Done!

File: src/Routes/AppRoute.js (hunk 1 above)
Comment (barunojha) [reply src/Routes/AppRoute.js]: Done!

**Pull request merged**